__tests__/endpoints/articles.test.js

```diff
@@ -128,6 +128,7 @@ describe("/api/articles/:article_id", () => {
                             created_at: "2020-07-09T20:11:00.000Z",
                             votes: 100,
                             article_img_url: "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+                            comment_count: 11,
```

Comment:
Ideally, previous tests shouldn't fail when new features are added and a new test should be created to test for new features, if being very picky